extensions/wikia/PhalanxII/templates/PhalanxStatsSpecial_index.php

```diff
@@ -1,10 +1,41 @@
-<?= $table ?>
-
-<? if ( isset( $editUrl ) ): ?>
-<a href="<?= htmlspecialchars($editUrl) ?>" class="modify"><?= wfMsg('phalanx-link-modify') ?></a>
-<? endif ?>
+<table class="wikitable phalanx-stats-table">
+	<thead>
+		<tr>
+			<th><?= wfMessage( 'phalanx-stats-table-id' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-user' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-type' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-create' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-expire' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-exact' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-regex' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-case' )->escaped(); ?></th>
+			<th><?= wfMessage( 'phalanx-stats-table-language')->escaped(); ?></th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+		<?php foreach ( $firstRow as $entry ): ?>
+			<td><?= htmlspecialchars( $entry ); ?></td>
+		<?php endforeach; ?>
+		</tr>
+		<tr>
+			<th><?= wfMessage( 'phalanx-stats-table-text' )->escaped(); ?></th>
+			<td colspan="8"><?= htmlspecialchars( $text ); ?></td>
+		</tr>
+		<tr>
+			<th><?= wfMessage( 'phalanx-stats-table-reason' )->escaped(); ?></th>
+			<td colspan="8"><?= $reason ?></td>
```

Comment:
Yeah, it wasn't escaped before so was vulnerable to stored XSS. The parsing was added after this comment in https://github.com/Wikia/app/pull/11161/commits/92e29d2a8c8a82334a1a2effe9eb7b5fcded9548 `getText` there is called after the `parse` function. The wikitext to HTML parser is safe to output, so it's fine now. :)